tmd/Soma/Soma.py

```diff
@@ -1,17 +1,19 @@
 '''
 tmd class : Soma
 '''
+import copy
+import numpy as np
 
 
 class Soma(object):
     '''Class of neuron soma
     '''
-    import numpy as _np
+    # pylint: disable=import-outside-toplevel
     from tmd.Soma.methods import get_center
     from tmd.Soma.methods import get_diameter
 
-    def __init__(self, x=_np.array([]), y=_np.array([]), z=_np.array([]),
-                 d=_np.array([])):
+    def __init__(self, x=np.array([]), y=np.array([]), z=np.array([]),
```

Comment:
Careful here. If you create 5 different somata they will all share the same numpy arrays!

Reply:
This did not change from previous version (I think) I just cleaned up the _